### PR TITLE
release-schema.json: add numbers and forReduction properties

### PR DESCRIPTION
**Pull request by odscjen**
I've used the `AwardCriterionNumber` object from the [award criteria](https://extensions.open-contracting.org/en/extensions/awardCriteria/master/schema/#awardcriterionnumber) extension here to add a `numbers` object. This has however added an additional field that we don't need for eForms, `numbers.fixed`. But even if eForms doesn't need it it seems like a field that's okay to have in the extension as another jurisdiction may have a need for it?

### CODE REVIEW

File: README.md
Comment (duncandewhurst) [inline README.md]: I don't think we should remove the reference to OCDS for the EU as the extension is still relevant in that context.

File: extension.json
Comment (duncandewhurst) [inline extension.json:26]: The award criteria extension should be in `dependencies` since this extension `$ref`'erences a definition from it, per the [handbook](https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html#create-an-extension).

File: release-schema.json
Comment (duncandewhurst) [inline release-schema.json]: If we want to reuse the `AwardCriterionNumber` definition, I think we'll need to update the code descriptions in `criterionThreshold.csv` to cover requests to participate (currently, they only mention bids) and we should probably rename the definition to `CriterionNumber`, too.

File: README.md
Comment (duncandewhurst) [inline README.md:13]: I think these examples would benefit from fleshing out with realistic values, rather than just the descriptions of the fields, and with an introductory sentence like those in the [award criteria extension](https://extensions.open-contracting.org/en/extensions/awardCriteria/master/).

File: README.md
Comment (duncandewhurst) [inline README.md]: Does it make sense to only have a criteria with a 30% weighting? I would've expected the percentages to add up to 100%, like the examples in the award criteria extension.

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

File: README.md
Comment (odscjen) [reply README.md]: fair enough, I'll add that back in. (and attempt to remember not to remove it from future changes)

File: extension.json
Comment (odscjen) [reply extension.json:26]: Ah! I'll change that, so it shouldn't be in `testDependencies` at all?

File: README.md
Comment (odscjen) [reply README.md]: good point, I'll change this to 100% instead of 30%

File: README.md
Comment (odscjen) [reply README.md:13]: agreed, I've attempted this in the new PR to the best of my understanding based on the eForms examples.

File: release-schema.json
Comment (odscjen) [reply release-schema.json]: Agreed, I'll create a new issue in the extensions repo for this. https://github.com/open-contracting/ocds-extensions/issues/199

File: extension.json
Comment (duncandewhurst) [reply extension.json:26]: According to the handbook, only if this extension adds fields to a definition from the award criteria extension, so I think that can be removed from `testDependencies`.

File: release-schema.json
Comment (duncandewhurst) [reply release-schema.json]: I think that we should make those changes to the award criteria extension before merging this PR, to avoid this extension being in an incoherent state.

Comment (duncandewhurst) [reply release-schema.json]: Perhaps worth adding it to the [tracker](https://docs.google.com/spreadsheets/d/1VIAFbuTCF1kMhP7ngWnlFrE9HXSUb6I6W8emlkUy3K8/edit#gid=1619313430) with a note, since the changes relate to eForms.

**Comment by duncandewhurst** [comment]
@odscjen checking that we're on the same page here - this PR is on hold until the updates to the award criteria extension are done, right?

**Comment by odscjen** [comment]
@duncandewhurst yes that's my understanding too

**Comment by jpmckinney** [comment]
Noting that this PR should now be unblocked.

**Review by duncandewhurst — APPROVED** [review]

**Comment by jpmckinney** [comment]
@yolile Do you know anyone using the selectionCriteria extension? This adds the awardCriteria extension as a dependency, so any publisher using selectionCriteria without awardCriteria will cause an error when patching the release schema with extensions.

File: release-schema.json
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "title": "Numbers",
          "description": "Numbers linked to the criterion.",
```

File: README.md
Comment (jpmckinney) [inline README.md]:
```suggestion
* Add the `forReduction` and `numbers` fields to `SelectionCriterion`. The `numbers` field reuses the `CriterionNumber` definition from the [Award criteria breakdown](https://extensions.open-contracting.org/en/extensions/awardCriteria/master/) extension.
```

File: release-schema.json
Comment (jpmckinney) [inline release-schema.json]: Follow style guide. Use OCDS, not EU, terms.

```suggestion
          "description": "Whether the criterion is used to select the potential suppliers to be invited to the second stage of the contracting process, if the number of invitations is limited.",
```

**Comment by jpmckinney** [comment]
> @yolile Do you know anyone using the selectionCriteria extension? This adds the awardCriteria extension as a dependency, so any publisher using selectionCriteria without awardCriteria will cause an error when patching the release schema with extensions.

It looks like only UK and Scotland are using this extension (and they also use award criteria breakdown).

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**
